Ryujinx.HLE/HOS/Services/Fs/IFileSystemProxy.cs

```diff
@@ -246,6 +268,18 @@ public ResultCode OpenPatchDataStorageByCurrentProcess(ServiceCtx context)
             return ResultCode.Success;
         }
 
+        [Command(400)]
+        // OpenDataStorageByCurrentProcess() -> object<nn::fssrv::sf::IStorage> dataStorage
+        public ResultCode OpenDeviceOperator(ServiceCtx context)
+        {
+            Result rc = _baseFileSystemProxy.OpenDeviceOperator(out LibHac.FsService.IDeviceOperator deviceOperator);
```

Comment:
```suggestion
            Result resultCode = _baseFileSystemProxy.OpenDeviceOperator(out LibHac.FsService.IDeviceOperator deviceOperator);
```
and add an empty line after.